src/outputgeneration/ParseTreeWriter.js

```diff
@@ -1501,7 +1501,7 @@ export class ParseTreeWriter extends ParseTreeVisitor {
    * @private
    */
   writelnList_(list, delimiter) {
```

Comment:
I actually did that first but changed it to make the param explicit. With explicit there is no arguments adaptor needed and the function stays monomorphic.